packages/codemods/src/codeMods/tests/command.test.ts

```diff
@@ -1,47 +1,30 @@
 import { CommandParser, CommandParserResult } from '../../command';
 
 describe('command parser', () => {
-  // TODO add back once we need command parser to do more things
-  // describe("help and usage", () => {
-  // it("prints a help message when run without arguments", () => {
-  // let message = "";
-  // const log = (msg: string) => {
-  // message = msg;
-  // };
-  // new CommandParser(log).parseArgs(["node", "foo"]);
-  // expect(message).toContain("Usage");
-  // });
-
-  // it('prints a help message when run without "--help"', () => {
-  // let message = "";
-  // const log = (msg: string) => {
-  // message = msg;
-  // };
-  // new CommandParser(log).parseArgs(["node", "foo", "--help"]);
-  // expect(message).toContain("Usage");
-  // });
+  describe('when called with a single argument', () => {
+    let result: CommandParserResult;
 
-  // it("directs the program to exit when run without arguments", () => {
-  // const result = new CommandParser(() => {}).parseArgs(["node", "foo"]);
-  // expect(result.shouldExit).toBeTruthy();
-  // });
-  // });
+    beforeAll(() => {
+      result = new CommandParser().parseArgs(['node', 'foo', 'bar/baz']);
+    });
 
-  describe('when called with a single argument', () => {
+    it('does not direct the program to exit', () => {
+      expect(result.shouldExit).toBeFalsy();
+    });
+  });
+  describe('parses string filters into an array', () => {
     let result: CommandParserResult;
 
     beforeAll(() => {
-      result = new CommandParser(() => {
-        return undefined;
-      }).parseArgs(['node', 'foo', 'bar/baz']);
+      result = new CommandParser().parseArgs(['node', 'foo', 'bar/baz', '--sf', 'one', 'two']);
     });
 
-    it('returns the path when called with a single argument', () => {
-      expect(result.path).toEqual('bar/baz');
+    it('has two strings parsed', () => {
+      expect(result.stringFilter.orElse([]).length).toEqual(2);
     });
 
-    it('does not direct the program to exit', () => {
-      expect(result.shouldExit).toBeFalsy();
+    it('first string is one', () => {
+      expect(result.stringFilter.then(v => v[0]).orElse('')).toEqual('one');
```

Comment:
Any reason this is `stringFilter.then` instead of `stringFilter.map` ?

Reply:
StringFilter is a maybe. Maybe doesn't have a straight map/bind yet.